templates/incl-editor-and-output.html

```diff
@@ -119,8 +119,8 @@
       <!-- tabindex=0 makes the div focusable -->
       <div id="output" tabindex=0 class="flex-1 rounded p-2 px-3 bg-gray-900 color-white w-full text-lg overflow-auto" style="min-height: 3rem"></div>
       <div id="turtlecanvas" class="flex-0 ltr:pl-4 rtl:pr-4"></div>
-      <div id="inline-modal" class="flex-0" style="display: none">
-        <div class="py-2 text-left px-3 border-4 border-teal-500 mt-3 rounded bg-green-100">
+      <div id="inline-modal" class="flex-0">
+        <div class="ask-modal py-2 text-left px-3 border-4 border-teal-500 mt-3 rounded bg-green-100" style="display: none">
```

Comment:
Please define the `ask-modal` as an `id` instead of a `class`. The modal itself is unique and should by HTML design be called by their id (as classes are by design used for elements that share a class).

Reply:
[Change class to id](https://github.com/hedyorg/hedy/pull/3668/commits/5da051dc60a3d17db45a904f07156b8fa4074125) fixes this.